example/init

```diff
@@ -150,11 +150,11 @@ riverctl border-color-unfocused 0x586e75
 # Set keyboard repeat rate
 riverctl set-repeat 50 300
 
-# Make all views with an app-id that starts with "float" and title "foo" start floating.
-riverctl rule-add float -app-id 'float*' -title 'foo'
+# Make all views with an app-id that starts with "float"
+riverctl float-filter-add app-id 'float*'
```

Comment:
Note: filtering on both `app-id` and `title` simultaneously does not seem to be possible anymore